Stack/Opc.Ua.Core/Stack/Client/SessionChannel.cs

```diff
@@ -102,5 +104,136 @@ public partial class SessionChannel
             return channel;
         }
         #endregion
+
+        #region Public Methods
+        /// <summary>
+        /// Creates a secure channel to the specified endpoint.
+        /// </summary>
+        /// <param name="configuration">The application configuration.</param>
+        /// <param name="connection">The client endpoint for the reverse connect.</param>
+        /// <param name="endpoint">A configured endpoint to connect to.</param> 
+        /// <param name="updateBeforeConnect">Update configuration based on server prior connect.</param>
+        /// <param name="checkDomain">Check that the certificate specifies a valid domain (computer) name.</param>
+        /// <returns>A <see cref="Task"/> representing the asynchronous operation.</returns>
+        public static async Task<ITransportChannel> CreateAsync(
```

Comment:
Hi @Zigby, could this function be in the client library? 
Thats really client implementation.

Reply:
The libraries handle all the stuff with certs, not core. Maybe name it just CreateChannelAsync.